vllm/vllm-readme.md

```diff
@@ -0,0 +1,5 @@
+# Optimize LLM serving with vLLM on Intel® GPUs
```

Comment:
why did you changed main `README.md` to `vllm-readme.md`? it doesn render anything by default. Can we change this file name to README.md?

Reply:
Also if you want to keep version specific READMEs then can you add links in this main readme for better user experience?